spring-integration-ftp/src/main/java/org/springframework/integration/ftp/dsl/Ftp.java

```diff
@@ -121,13 +123,36 @@ public static FtpMessageHandlerSpec outboundAdapter(RemoteFileTemplate<FTPFile>
 	 * @param remoteFileTemplate the remote file template.
 	 * @param fileExistsMode the file exists mode.
 	 * @return the spec.
+	 * @deprecated in favor of {@link #outboundAdapter(FtpRemoteFileTemplate, FileExistsMode)}
 	 */
+	@Deprecated
 	public static FtpMessageHandlerSpec outboundAdapter(RemoteFileTemplate<FTPFile> remoteFileTemplate,
 			FileExistsMode fileExistsMode) {
 
 		return new FtpMessageHandlerSpec(remoteFileTemplate, fileExistsMode);
 	}
 
+	/**
+	 * A {@link FtpMessageHandlerSpec} factory for an outbound channel adapter spec.
+	 * @param ftpRemoteFileTemplate the remote file template.
+	 * @return the spec.
```

Comment:
Please, add `@since 5.4` for all new public API

Reply:
Added 5.4 version to new methods